src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int Figure_Count = 5;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int Max_Value = 25;
```

Comment:
Please follow the rules of constants naming:
```suggestion
    private static final int MAX_VALUE = 25;
```